File: 例子9-2 RPC 聊天室 双工/TalkRPC/Server/UserInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using ZYSocket.share;

namespace Server
{
    public class UserInfo
    {
        public string UserName { get; set; }
        public ZYNetRingBufferPoolV2 Stream { get; set; }
        public System.Net.Sockets.SocketAsyncEventArgs Asyn { get; set; }

        public UserInfo(System.Net.Sockets.SocketAsyncEventArgs asyn)
        {
            this.Asyn = asyn;
            Stream = new ZYNetRingBufferPoolV2(1024 * 1024 * 4);
        }
    }

}

File: TestServer-IronPython/TestConnect/UserInfo.py
import clr;
clr.AddReference("ZYSocketShare");
from ZYSocket.share import  *;
from System import *

class UserInfo(Object): 
  
    def __init__(self,asyn,size):
        self.Stream =ZYNetBufferReadStreamV2(size);
        self.Asyn=asyn;

#    def __init__(self,**kw):     
#        super(UserInfo,self).__init__(**kw);



       

  



